js/frames/groupShowcase-5.js

```diff
@@ -1,6 +1,7 @@
 loadFramePacks([
 	{name:'Set Showcase Frames (Chronological)', value:'disabled'},
 	{name:'Paranormal (DSK)', value:'Paranormal'},
+	{name:'Japanese (DSK)', value:'JapaneseShowcase'},

```

Comment:
It sounds like Wizards will be re-using this frame going forward. It won't be exclusive to Duskmourn. Can we group this next to the borderless cards instead of with the other showcase cards?